src/ray/common/ray_config_def.h

```diff
@@ -253,6 +253,8 @@ RAY_CONFIG(uint32_t, object_store_get_max_ids_to_print_in_warning, 20)
 RAY_CONFIG(int64_t, gcs_service_connect_retries, 50)
 /// Waiting time for each gcs service connection.
 RAY_CONFIG(int64_t, internal_gcs_service_connect_wait_milliseconds, 100)
+/// The duration between detect gcs.
+RAY_CONFIG(int64_t, gcs_detect_timeout_milliseconds, 100)
```

Comment:
maybe this can be more accurate. Essentially this is an interval for heartbeat, not a timeout that would treat redis as failed. 

How about  `gcs_redis_heartbeat_interval_milliseconds` ?

Reply:
fixed